README.md

```diff
@@ -274,16 +288,18 @@ Now, create the pipelinerun configuration.
 ```
 
 It will create pipeline with a below message on terminal.
+
 ```
   pipelinerun.tekton.dev/application-pipeline-run created
 ```
 
 To check the status of the pipeline created:
+
 ```
   kubectl describe pipelinerun application-pipeline-run
 ```
 
-You may need to re-run this command based on the status. It shows the interim status as:
+You may need to rerun this command based on the status. It shows the interim status as:<!--EM: I'm confused about this use of HTML in this following listing and the ones below. Should be in a code listing to show that it's output? Am I misunderstanding what exactly this is?-->
 
```

Comment:
Right, it can be code listing.

But I wanted to highlight(bold) some of the text, hence I used the html tags. If there is any other way to highlight the content in code block, we can use that.

Reply:
Sadly, there's not a way to highlight text in code listings right now. we'll just leave it as is and go back and revisit at another time.

Reply:
ok